triggers/run-completion-event-trigger/cmd/main.go

```diff
@@ -55,3 +58,17 @@ func main() {
 	}
 
 }
+
+func unaryLoggerInterceptor(baseLogger logr.Logger) grpc.UnaryServerInterceptor {
+	return func(
+		ctx context.Context,
+		req any,
+		info *grpc.UnaryServerInfo,
+		handler grpc.UnaryHandler,
+	) (resp any, err error) {
+		logger := baseLogger.WithValues("grpc_method", info.FullMethod)
+		ctx = logr.NewContext(ctx, logger)
```

Comment:
```suggestion
		ctx = logr.NewContext(ctx, baseLogger)
```

having the grpc_method on every log line triggered within a given request is going to be overly noisy for a service with only one endpoint anyway

Reply:
https://github.com/sky-uk/kfp-operator/pull/627/commits/d4425cf03cb746ecb41453053c7c334191aa72bf